### PR TITLE
Add conv_transpose and sigmoid op

### PR DESCRIPTION
**Pull request by wjj19950828**
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

## Do follow Contributes
1、Add Conv2DTranspose and Sigmoid op
2、Add test cases

### CODE REVIEW

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: tests/python/frontend/paddlepaddle/test_forward.py [hunk 1]
```diff
@@ -321,6 +321,50 @@ def forward(self, inputs):
...
+            self.conv_transpose = nn.Conv2DTranspose(3, 5, 3)
+
+        @paddle.jit.to_static
+        def forward(self, inputs):
+            return self.conv_transpose(inputs)
+
+    class Conv2DTranspose2(nn.Layer):
+        def __init__(self):
+            super(Conv2DTranspose2, self).__init__()
+            self.conv_transpose = nn.Conv2DTranspose(
```
[the remark this answers is not in the document]
Comment (wjj19950828) [reply tests/python/frontend/paddlepaddle/test_forward.py:341]: Done.

File: tests/python/frontend/paddlepaddle/test_forward.py [hunk 2]
```diff
@@ -321,6 +321,50 @@ def forward(self, inputs):
     verify_model(Conv2D2(), input_data=conv2d_input_data)
 
 
+@tvm.testing.uses_gpu
+def test_forward_conv_transpose():
+    # Note we do not test with groups  > 1 because that is not supported
+    # in tvm for conv transpose operations
```
Comment (jiangjiajun) [inline tests/python/frontend/paddlepaddle/test_forward.py:327]: TVM不支持groups>1？

Comment (wjj19950828) [reply tests/python/frontend/paddlepaddle/test_forward.py:327]: 对的，在pytorch的单测中也有指出这个

Comment (jiangjiajun) [reply tests/python/frontend/paddlepaddle/test_forward.py:327]: 如果groups大于1的时候会怎样

Comment (wjj19950828) [reply tests/python/frontend/paddlepaddle/test_forward.py:327]: 在我将groups设置成3之后跑上述第一个单测会导致维度对不上，如下图
image
